### ISSUE TITLE
isDragging compareEpsilon should remember if drag started

### ISSUE THREAD

**Issue by Irev-Dev** [KittyCAD/modeling-app#1410]
`isDragging` is determined entirely by if your mouse is far enough away from the starting point, which means you can't drag the mouse away and then back close to where you started.
~`src/clientSideScene/sceneInfra.ts:765`

https://github.com/KittyCAD/modeling-app/assets/29681384/09a089b9-a2c8-471b-836c-fddd5ffd5f5c

**Comment by franknoirot** [comment]
Giving some visibility to @andrewvarga about this, seems like you'd see right through this

**Comment by andrewvarga** [comment]
Yeah I'll look at this one! There is a similar issue I found recently which is this code:

https://github.com/KittyCAD/modeling-app/blob/78fcfca2dde48fce89251625c77c38bf34814847/src/clientSideScene/sceneEntities.ts#L2944-L2946

Here we ignore movements above a certain threshold but this threshold is used in world space, not screen space pixels, so when you zoom out a lot you only need to move it a bit to exceed the limit..do you remember which bug this was fixing (cc @Irev-Dev )? If we keep it, I suggest we convert that check to screen space, so any 100pixel mouse move jumps can be ignored..
